src/Crowdin.Api/Bundles/BundlesApiExecutor.cs

```diff
@@ -1,47 +1,62 @@
-﻿
-using System.Collections.Generic;
-using System.Net;
-using System.Threading.Tasks;
-using JetBrains.Annotations;
 
 using Crowdin.Api.Core;
 using Crowdin.Api.SourceFiles;
+using JetBrains.Annotations;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
 
 namespace Crowdin.Api.Bundles
 {
     public class BundlesApiExecutor
     {
         private readonly ICrowdinApiClient _apiClient;
         private readonly IJsonParser _jsonParser;
-        
+
         public BundlesApiExecutor(ICrowdinApiClient apiClient)
         {
             _apiClient = apiClient;
             _jsonParser = apiClient.DefaultJsonParser;
         }
-        
+
         public BundlesApiExecutor(ICrowdinApiClient apiClient, IJsonParser jsonParser)
         {
             _apiClient = apiClient;
             _jsonParser = jsonParser;
         }
-        
+
         /// <summary>
         /// List bundles. Documentation:
-        /// <a href="https://developer.crowdin.com/api/v2/#operation/api.projects.bundles.getMany">Crowdin API</a>
-        /// <a href="https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.bundles.getMany">Crowdin Enterprise API</a>
+        /// <a href="https://developer.crowdin.com/api/v2/string-based/#operation/api.projects.bundles.branches.getMany">Crowdin API</a>
+        /// <a href="https://developer.crowdin.com/enterprise/api/v2/string-based/#operation/api.projects.bundles.branches.getMany">Crowdin Enterprise API</a>
```

Comment:
Wrong links. It shouldn't point to the Bundle List Branches API. It's probably inadvertently linked with the method below

Reply:
Updated